src/pages/City/City/City.tsx

```diff
@@ -466,6 +466,8 @@ const City = () => {
           const check = await getCheckPlastMember(admin.userId);
           if(check.data){
             await addCityAdmin(admin);
+            admins.push(admin);
```

Comment:
What if `addCityAdmin` fails? You will still display that a new admin was added. I suggest fetching admins from server, or at least use `.then()`.

Reply:
ok, ty